docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -1062,10 +1097,10 @@ subquestion: |
   ${ action_button_html(url_action('button_uninstall_package'),label="Uninstall interview", color="danger") }
 
   Download the YAML file only:  
-  [:download: Download YAML](${interview_download.url_for(attachment=True)})
+  [:download: Download YAML](${interview_download})
```

Comment:
Are these changes intentional? Not sure I understand it.

Reply:
I switched from using `def create_package_zip` to using the core function `zip_file` for multiple-attachments purpose, which doesn't need `url_for`.

Reply:
Here is the difference I saw:

The download link with `url_for`
https://da-dev.masslegalhelp.org/**uploadedfiledownload**/13161/docassemble-lalala2.yml

without it
https://da-dev.masslegalhelp.org/**uploadedfile**/13161/docassemble-lalala2.yml

So the url is slightly better with it, because it has the word `download`, and `uploadedfile `is not right in our case. 

Any other benefits?